pkg/sfu/utils/rangemap.go

```diff
@@ -63,6 +76,21 @@ func NewRangeMap[RT rangeType, VT valueType](size int) *RangeMap[RT, VT] {
 	return r
 }
 
+func (r *RangeMap[RT, VT]) MarshalLogObject(e zapcore.ObjectEncoder) error {
+	e.AddInt("numRanges", len(r.ranges))
+
+	// just the last 10 ranges max
+	startIdx := len(r.ranges) - 10
+	if startIdx < 0 {
+		startIdx = 0
+	}
+	for i := startIdx; i < len(r.ranges); i++ {
+		e.AddObject(fmt.Sprintf("range[%d]", i), r.ranges[i])
+	}
```

Comment:
Looping because of a missing feature in zapcore where slices of struct (i. e. non-primitive does not work when using `AddReflected`).

Also, logging only newest 10 so that it does not log too much in case the range map contains a bunch of entries.